worker/worker.go

```diff
@@ -277,12 +277,36 @@ const (
 //	           identifies group of workflow and activity implementations that are
 //	           hosted by a single worker process
 //	options  - configure any worker specific options like logger, metrics, identity
+//
+// DEPRCATED: use NewV2 instead since this implementation will panic on error
 func New(
```

Comment:
please update CHANGELOG.md about this new validation and the new `worker.NewV2` constructor

Reply:
sure